src/main/java/com/enderio/base/common/tag/EIOTags.java

```diff
@@ -109,6 +109,8 @@ private static void init() {}
         public static final TagKey<Item> BLOCKS_SOULARIUM = ItemTags.create(new ResourceLocation(NeoForgeVersion.MOD_ID, "storage_blocks/soularium"));
         public static final TagKey<Item> BLOCKS_VIBRANT_ALLOY = ItemTags.create(new ResourceLocation(NeoForgeVersion.MOD_ID, "storage_blocks/vibrant_alloy"));
 
+        public static final TagKey<Item> STORED_ENTITY = ItemTags.create(EnderIO.loc("stored_entity"));
```

Comment:
the name "STORED_ENTITY" doesn't make sense for the tag. Same with the attachment. Should be named something that conveys that item is capable of storage